src/pages/community/vision-and-mission.md

```diff
@@ -0,0 +1,21 @@
+<!-- Put each sentence on a single line to make reviewing easier -->
+# OpenInfra Labs Vision and Mission
+                                                                     
+This page is to help you understand the ovearching purpose (vision) of the OpenInfra Labs,, and how we're bringing about that vision, which is our mission.
+
+## Vision of OpenInfra Labs
+
+In support of the [Four Opens](https://openinfra.dev/four-opens/), OpenInfra Labs vision is the creation and sustainability of all-Open Infrastructure from myriad all-Open tools, platforms, and methods.
+This Open Infrastructure enables the practice of Open Source software development and other [Open Work](https://fossrit.github.io/open-work-definition/) collaborations.
+
+## Mission of OpenInfra Labs
+
+In fulfilling the OpenInfra Labs vision, Laboratory projects work to define key concepts for establishing current and future Open Infrastructures.
```

Comment:
It seems like more than just key concepts.

"Open Infra Labs exists as a collaborative space where a, b, c"?